packages/hash/api/src/db/postgres/link/sql/link_versions.util.ts

```diff
@@ -87,6 +87,7 @@ export const updateLinkVersionRow = async (
     where
       source_account_id = ${params.sourceAccountId}
       and link_version_id = ${params.linkVersionId}
+    returning *;
```

Comment:
We need this query to return something, because `sql.one` helps us verify a single record has been updated as expected.